app/mailers/donations_mailer.rb

```diff
@@ -1,16 +1,19 @@
 class DonationsMailer < ApplicationMailer
   def thank_you_email(donation)
     @mobilization = donation.widget.mobilization
+    @widget = donation.widget
 
+    ws = @widget.settings
     user = @mobilization.user
-    sender = user.first_name
-    email_address = user.email
+    sender = ws['sender_name'] ? ws['sender_name'] : user.first_name
```

Comment:
Not reeeally necessary, but as a reminder, you could use `#presence` to do something like:

`sender = ws['sender_name'].presence || user.first_name`.